src/main/java/Leas_Liudmila/Main.java

```diff
@@ -1,4 +1,4 @@
-package Leas_Liudmila;
+package src.main.java.Leas_Liudmila;
```

Comment:
this package is wrong
![image](https://github.com/dgrimaliuc/tekwill-qa-automation/assets/116957079/58c7ced1-6126-4317-892f-cc6ee3b312f5)
please use just Leas_Liudmila instead, wirhout src.main...

Reply:
now your pakage says to JVM that this file should be located in the last package 
![image](https://github.com/dgrimaliuc/tekwill-qa-automation/assets/116957079/3a7694d7-f0df-4d19-911a-7ae81133fc60)